cpp/test/prims/batched/batched_matrix.h

```diff
@@ -17,7 +17,8 @@
 #pragma once
```

Comment:
nitpick: since this file is already under `batched` folder, maybe just call this one as `matrix.h`?